backend/dal/Migrations/v01.16.00/Up/Postup/01-ViewProperties.sql

```diff
@@ -183,7 +183,7 @@ LEFT JOIN dbo.[ParcelBuildings] pb ON b.[Id] = pb.[BuildingId]
 LEFT JOIN dbo.[Parcels] p ON pb.[ParcelId] = p.[Id]
 JOIN dbo.[PropertyClassifications] c ON b.[ClassificationId] = c.[Id]
 LEFT JOIN dbo.[Users] u ON b.UpdatedById = u.Id
-LEFT JOIN dbo.[Agencies] a ON b.[AgencyId] = a.[Id]
```

Comment:
hmm.....even though I renamed my Postup to PostUp, the repo isn't detecting it for some reason?